packages/schematics/angular/component/schema.json

```diff
@@ -48,6 +48,13 @@
       "alias": "t",
       "x-user-analytics": 10
     },
+    "standalone": {
+      "description": "Whether the generated component is standalone.",
+      "type": "boolean",
+      "default": false,
+      "alias": "sa",
```

Comment:
POSIX compliant aliases should only be a single letter.  This is not currently enforced but will be in the future.
Since lowercase `s` is already taken, we could use capital `S`, a different letter (`a`?), or drop the alias for now.

Reply:
I'll skip the alias for now.